sunpy/net/vso/vso.py

```diff
@@ -47,7 +47,9 @@
 TIME_FORMAT = config.get("general", "time_format")
 
 DEFAULT_URL_PORT = [{'url': 'http://docs.virtualsolar.org/WSDL/VSOi_rpc_literal.wsdl',
-                     'port': 'nsoVSOi', 'transport': WellBehavedHttpTransport}]
+                     'port': 'nsoVSOi', 'transport': WellBehavedHttpTransport},
+                    {'url': 'https://sdac.virtualsolar.org/API/VSOi_rpc_literal.wsdl',
```

Comment:
Does 0.9 have zeep? I am pretty sure this is a zeep only thing.

Reply:
You mean using that specific wsdl is a zeep feature? 0.9 does not have zeep.

Reply:
Yeah I don't think that WDSL will work with suds-jurko because of missing https features.

Reply:
Ok I will undo those commits.